easybuild/easyblocks/s/siesta.py

```diff
@@ -208,26 +208,13 @@ def configure_step(self):
 
             elsi = get_software_root('ELSI')
             if elsi:
-                elsi_libs = ['elsi', 'fortjson', 'OMM', 'MatrixSwitch', 'NTPoly']
+                if not os.path.isfile(os.path.join(elsi, 'lib', 'libelsi.so')):
```

Comment:
Use get_shared_lib_ext from easybuild.tools.systemtools instead of hardcoding .so (see gromacs.py for example if needed)

Reply:
my bad